apps/meteor/server/publications/messages.ts

```diff
@@ -173,11 +189,89 @@ export async function handleCursorPagination(
 	}
 
 	return {
-		[type.toLowerCase()]: response,
+		updated: type === 'UPDATED' ? response : [],
+		deleted: type === 'DELETED' ? response : [],
```

Comment:
Do we need the refactor? Or can we just create more detailed types to cover what was there before? 👀